provisioning_templates/provision/kickstart_default.erb

```diff
@@ -87,7 +88,7 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
-<% if (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) -%>
+<% if use_ntp -%>
```

Comment:
Providing an ntp server is not supported by chrony?

Reply:
`--ntpservers` wasn't added to Anaconda until F16 when Chrony was made the default, so this is the opposite case, we don't supply --ntpservers to timezone unless using chrony.

Reply:
Yes, but if the `use-ntp` param is set to true, those parameters are no longer passed in while they were without it. This line probably needs to stay unchanged.

Reply:
I don't follow, we never passed `--ntpservers` to timezone until this PR

Reply:
I think you're looking at the wrong diff, i.e. the diff between my two commits, vs. the diff w/ develop.

Reply:
You are correct.